src/Toolkit/Toolkit.WinUI/Themes/SharedResources.xaml

```diff
@@ -0,0 +1,20 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+    <FontFamily x:Key="CalciteIconFont">ms-appx:///Esri.ArcGISRuntime.Toolkit.WinUI/Assets/calcite-ui-icons-24.ttf#calcite-ui-icons-24</FontFamily>
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Default">
+            <BitmapImage x:Key="MeasureLengthImage" UriSource="ms-appx:///Esri.ArcGISRuntime.Toolkit.WinUI/Assets/MeasureLength.png" />
+            <BitmapImage x:Key="MeasureAreaImage" UriSource="ms-appx:///Esri.ArcGISRuntime.Toolkit.WinUI/Assets/MeasureArea.png" />
+            <BitmapImage x:Key="MeasureFeatureImage" UriSource="ms-appx:///Esri.ArcGISRuntime.Toolkit.WinUI/Assets/MeasureFeature.png" />
+        </ResourceDictionary>
+        <ResourceDictionary x:Key="Light">
+            <BitmapImage x:Key="MeasureLengthImage" UriSource="ms-appx:///Esri.ArcGISRuntime.Toolkit.WinUI/Assets/MeasureLength.png" />
+            <BitmapImage x:Key="MeasureAreaImage" UriSource="ms-appx:///Esri.ArcGISRuntime.Toolkit.WinUI/Assets/MeasureArea.png" />
+            <BitmapImage x:Key="MeasureFeatureImage" UriSource="ms-appx:///Esri.ArcGISRuntime.Toolkit.WinUI/Assets/MeasureFeature.png" />
+        </ResourceDictionary>
+        <ResourceDictionary x:Key="Dark">
+            <BitmapImage x:Key="MeasureLengthImage" UriSource="ms-appx:///Esri.ArcGISRuntime.Toolkit.WinUI/Assets/MeasureLengthDark.png" />
+            <BitmapImage x:Key="MeasureAreaImage" UriSource="ms-appx:///Esri.ArcGISRuntime.Toolkit.WinUI/Assets/MeasureAreaDark.png" />
+            <BitmapImage x:Key="MeasureFeatureImage" UriSource="ms-appx:///Esri.ArcGISRuntime.Toolkit.WinUI/Assets/MeasureFeatureDark.png" />
+        </ResourceDictionary>
+    </ResourceDictionary.ThemeDictionaries>
```

Comment:
Aren't these only used by the measure tool? If so, only declare those in its xaml resource file.